pavelib/acceptance_test.py

```diff
@@ -13,22 +14,23 @@
 __test__ = False  # do not collect
 
 
-@task
 @needs(
     'pavelib.prereqs.install_prereqs',
     'pavelib.utils.test.utils.clean_reports_dir',
 )
 @cmdopts([
     ("system=", "s", "System to act on"),
-    ("default_store=", "m", "Default modulestore to use for course creation"),
+    ("default-store=", "m", "Default modulestore to use for course creation"),
     ("fasttest", "a", "Run without collectstatic"),
-    ("extra_args=", "e", "adds as extra args to the test command"),
     make_option("--verbose", action="store_const", const=2, dest="verbosity"),
     make_option("-q", "--quiet", action="store_const", const=0, dest="verbosity"),
     make_option("-v", "--verbosity", action="count", dest="verbosity"),
```

Comment:
@cpennington the other question I had was about verbosity. It looks to me like that also just passes through as an option to manage.py, with --verbose and --quiet being shortcuts (with priority among them, if multiple are used, that is probably not clear to the end user).

Totally understand if this is expanding the scope of what you were trying to fix anyhow, if you want to get your changes merged in to get pudb working.